site/data/docs/custom-wallet-list.mdx

```diff
@@ -60,78 +61,75 @@ const App = () => (
 
 ### Built-in wallets
 
-The following wallets are provided via the `wallet` object (in alphabetical order).
+The following wallets are provided via the `wallet` object.
 
-#### Argent
+#### Generic Wallets
```

Comment:
Yeah, I really like this separation between wallets: "Generic" vs. "Specific". Nice.